tasks/bundle-api.js

```diff
@@ -21,7 +21,7 @@ async function bundleAPI() {
                 tsconfigOverride: {
                     compilerOptions: {
                         removeComments: true,
-                        target: 'es5',
+                        target: 'ES6',
```

Comment:
This change is not related to iframes. I didn't analyze how the browser market changed in the meantime.

Reply:
well.. your new array.ts use ES6 :)